src/app/news/page.tsx

```diff
@@ -13,20 +13,10 @@ export default async function FbPage() {
         Najnowsze aktualności
       </h1>
       <hr className="w-full" />
-      {user == null || posts == null || posts.length === 0 ? (
-        <Alert className="w-1/2" variant="destructive">
-          <AlertTitle>Błąd Sieciowy</AlertTitle>
-          <AlertDescription>
-            Nie udało się pobrać postów z Facebooka. Spróbuj ponownie później.
-          </AlertDescription>
-        </Alert>
+      {user == null || posts == null ? (
```

Comment:
Tu może się trochę czepiam, niech mnie najwyżej Pola poprawi, ale ja jestem fanem strong type checków.
```suggestion
      {user === null || posts === null ? (
```

Reply:
ogólnie to zawsze powinno się używać `===`, co też zapewnia nasz `@solvro/config`, za wyjątkiem porównania z null-em, bo on tam wyłapuje `null` i `undefined` gdy używa się samego `==`, co jest zwykle żądane i nie wpływa negatywnie na kod